stylesheets/main.css

```
@@ -7,7 +7,7 @@
 
 	/* shuttle config */
 	--shuttle-logo: var(--shuttle-logo-nasa);
-	--rocket-fuel: 0;
+	--rocket-fuel: 10;
```

I think this should be set to 1. 

```suggestion
	--rocket-fuel: 1;
```